ggshield/core/ui/plain_text/plain_text_ggshield_ui.py

```diff
@@ -29,13 +73,17 @@ def create_message_only_scanner_ui(
         return PlainTextScannerUI()
 
     def create_progress(self, total: int) -> GGShieldProgress:
-        return PlainTextGGShieldProgress(self)
+        return PlainTextGGShieldProgress()
```

Comment:
I'm not sure what changed there and what `PlainTextGGShieldProgress` does ? :'(

Reply:
PlainTextGGShieldProgress was already there: it implements GGShieldProgress for the plaintext UI.

PlainTextGGShieldProgress inherits its `__init__()` from GGShieldProgress. This `__init__()` used to take a `GGShieldUI` instance before, [but it does not anymore][diff]. This change adapts the code to the new `__init__()` signature.

[diff]: https://github.com/GitGuardian/ggshield/pull/971/files/dff82c4287221e5f14cf12149e68953b589ecc56..88e89bd2f139d89f8a87f8f35bec73bb64a69d93#diff-f969bd1702588d982efb096f26b476484635bb6da3d3d8a913748a91e88c27bbL15